tools/scrape_optd_airlines_logos_from_websites.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import csv
+
+import requests
+
+successes = 0
+failures = 0
+with open('../opentraveldata/optd_airlines_websites_wkdt.csv', newline='') as csvfile:
+    rows = csv.reader(csvfile, delimiter='^')
+    for row in rows:
+        iata_code = row[1]
+        website = row[3]
+        if iata_code and website:
+            url = "https://logo.clearbit.com/"+website
```

Comment:
came across this https://blog.clearbit.com/logo/ and thought it could be interesting for this use case...